components/org.apache.stratos.manager.console/console/configure_form.jag

```diff
@@ -86,6 +86,15 @@ if(isEdit == false && elements.action != 'new') {
             }
             break;
 
+        case "deploymentpolicies":
+            formTitle = "Deployment Policie",
```

Comment:
Nope, this will properly handle inside the page. 

Deployment Policie -- missing "s" added by partial